.hound.yml

```diff
@@ -0,0 +1,4 @@
+fail_on_violations: true
+
+eslint:
+  version: 6
```

Comment:
@cedric add an empty line at the end of the file

Reply:
@johnkegz , Working on it👍🏾